app/lib/actions/katello/repository/clear.rb

```diff
@@ -11,7 +11,9 @@ def plan(repo)
            Pulp::Repository::RemoveFile,
            Pulp::Repository::RemovePuppetModule,
            Pulp::Repository::RemoveDockerManifest,
-           Pulp::Repository::RemoveDockerManifestList].each do |action_class|
+           Pulp::Repository::RemoveDockerManifestList,
+           Pulp::Repository::RemoveDockerTag,
```

Comment:
This also adds `RemoveDockerTag`. This seems like it's an extra change on this issue?

Reply:
The main intent of the clear call is to say "remove all units belonging to this repository". So I am retroactively including docker units also.